src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -417,6 +431,23 @@ def persist_flow_run(self, run_info: FlowRunInfo) -> None:
         )
         line_run_record.dump(self._run_infos_folder / filename)
 
+    def _load_all_flow_run_info(self, parse_const_as_str: bool = False) -> List[FlowRunInfo]:
+        flow_run_infos = []
+        for line_run_record_file in sorted(self._run_infos_folder.iterdir()):
+            new_runs = self._load_info_from_file(line_run_record_file, parse_const_as_str)
+            for new_run in new_runs:
+                new_run = resolve_multimedia_data_recursively(line_run_record_file, new_run)
+                flow_run_infos.append(new_run)
+                run_info = FlowRunInfo.deserialize(new_run)
+                line_number = run_info.index
+                self._loaded_flow_run_info[line_number] = run_info
+        return flow_run_infos
+
+    def load_flow_run_info(self, line_number: int = None) -> List[NodeRunInfo]:
```

Comment:
Should return object

Reply:
It seems like this function should return FlowRunInfo. According to the code:
```
        self._loaded_flow_run_info = {}  # {line_number: flow_run_info}
        self._loaded_node_run_info = {}  # {line_number: [node_run_info]}
```